indico/migrations/versions/20200804_1308_497c61b68050_fix_session_friendly_id_index.py

```diff
@@ -0,0 +1,29 @@
+"""Fix session friendly id index
+
+Revision ID: 497c61b68050
+Revises: 12fbf6af2367
+Create Date: 2020-08-04 13:08:57.032172
+"""
+
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = '497c61b68050'
+down_revision = '12fbf6af2367'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.execute('''
+        DROP INDEX events.ix_uq_sessions_friendly_id_event_id;
+        CREATE UNIQUE INDEX ix_uq_sessions_friendly_id_event_id ON events.sessions USING btree (friendly_id, event_id) WHERE NOT is_deleted;
+    ''')
+
+
+def downgrade():
+    op.execute('''
+        DROP INDEX events.ix_uq_sessions_friendly_id_event_id;
+        CREATE UNIQUE INDEX ix_uq_sessions_friendly_id_event_id ON events.sessions USING btree (friendly_id, event_id);
+    ''')
```

Comment:
I think this can all be done with the high-level alembic API to drop/create indexes.

(but that said, I'd also be tempted to just put SQL there since it's less work :P)

Reply:
FYI, I just cherry-picked this commit into master and amended it with the suggested change (71539618f4). that way we have less PRs with alembic revisions and it's something i'd like to have included when deploying next week, while i want to test this PR more thoroughly first :)

Reply:
Great, thanks a lot. I've rebased the PR now :)